raiden/tests/utils/eth_node.py

```diff
@@ -202,13 +201,11 @@ def geth_keystore(datadir: str) -> str:
     return os.path.join(datadir, "keystore")
 
 
-def geth_keyfile(datadir: str, address: Address) -> str:
+def geth_keyfile(datadir: str) -> str:
```

Comment:
can we make this a single function instead of two?

Reply:
which two functions do you refer to? Is it
https://github.com/raiden-network/raiden/blob/961df1977fa93d267060b41c0041f831730552c8/raiden/tests/utils/eth_node.py#L204-L207

vs

https://github.com/raiden-network/raiden/blob/961df1977fa93d267060b41c0041f831730552c8/raiden/tests/utils/eth_node.py#L282-L285

?

Reply:
I mean both of them, can we have a single function, say `eth_keyfile` that returns a path that can be used by either client?